### PR TITLE
Add some resources to handbook

### PR DESCRIPTION
**Pull request by jamescdavis**
## Purpose

Add some resources to the handbook.

## Summary of Changes

Added resources to the resources page for Ember, TypeScript, and Ember + TypeScript and re-linked in navigation.

## Side Effects / Testing Notes

n/a

## Ticket

n/a

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: lib/handbook/addon/docs/resources/template.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
 # Additional Resources
 
-TODO
+## EmberJS
+- [Ember.js Guides](https://guides.emberjs.com)
+- [Ember.js API Documentation](https://emberjs.com/api/ember/release)
+
+## TypeScript
+- [TypeScript in 5 minutes](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html) - get going fast
+- [TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/basic-types.html) - offical from Microsoft
```
Comment (aaxelb) [inline lib/handbook/addon/docs/resources/template.md]: s/offical/official

File: lib/handbook/addon/docs/template.hbs [hunk 2]
```diff
@@ -7,6 +7,7 @@
         {{nav.section 'Development'}}
         {{nav.item 'Quickstart' 'docs.quickstart'}}
         {{nav.item 'Troubleshooting' 'docs.troubleshooting'}}
+        {{nav.item 'Additional Resources' 'docs.resources'}}
```
Comment (aaxelb) [inline lib/handbook/addon/docs/template.hbs]: s/R/r

Comment (jamescdavis) [reply lib/handbook/addon/docs/template.hbs]: Should the title on the page also use this casing?

Comment (aaxelb) [reply lib/handbook/addon/docs/template.hbs]: No, page headers are title-case, at least going by osf-style: https://centerforopenscience.github.io/osf-style/language.html#capitalization

**Pull request merged**